### ISSUE TITLE
ng serve does not run on ubuntu 16.04

### ISSUE THREAD

**Issue by wk-done** [Microsoft/vscode-recipes#27]
Hi all,

I have the following environment
- OS: Ubuntu 16.04
- Visual Studio Code: Version 1.16.1
- VS Code - Debugger for Chrome 3.3.0

I want to debug a angular-cli.

I followed the guide [Chrome Debugging with Angular CLI](https://github.com/Microsoft/vscode-recipes/tree/master/Angular-CLI)
I just copied the content from launch.json
```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "name": "ng serve",
      "type": "chrome",
      "request": "launch",
      "url": "http://localhost:4200/#",
      "webRoot": "${workspaceRoot}"
    },
    {
      "name": "ng test",
      "type": "chrome",
      "request": "launch",
      "url": "http://localhost:9876/debug.html",
      "webRoot": "${workspaceRoot}"
    },
    {
      "name": "ng e2e",
      "type": "node",
      "request": "launch",
      "program": "${workspaceRoot}/node_modules/protractor/bin/protractor",
      "protocol": "inspector",
      "args": ["${workspaceRoot}/protractor.conf.js"]
    }      
  ]
}
```
then I started in the terminal`npm start`.

I tried to start the debugger by choosing *ng serve* and pressing on the "play"-button.
This lead to this error:
```
Error processing "launch": Error: Can't find Chrome - install it or set the "runtimeExecutable" field in the launch config.
```
Ok, I have the chromium-browser installed on my machine.
From a terminal window I can start it by calling `chromium-browser`

So I added the entry `"runtimeExecutable": "chromium-browser",` to my launch configuration:
here is the whole block:
```json
  {
    "name": "ng serve",
    "runtimeExecutable": "chromium-browser ",
    "type": "chrome",
    "request": "launch",
    "url": "http://localhost:4200/#",
    "webRoot": "${workspaceRoot}"
  }
```
Using this configuration I get the error `Attribute 'runtimeExecutable' does not exist ('chromium-browser ')`

I also tried to give the full path to the executable. It doesn't help. Same error:
`Attribute 'runtimeExecutable' does not exist ('/usr/bin/chromium-browser')`

Any idea what I'm doing wrong? Or what to check next?
Thanks in advance.

**Comment by auchenberg** [comment]
We try to launch `chrome` and if this binary isn't around or alias on Ubuntu you need to set your own `runtimeExecutable` which is a path.

Depending on where `chromium-browser` is installed, try running `which chromium-browser` and set the `runtimeExecutable` path.

**Comment by auchenberg** [comment]
On Unix we try to look for `/usr/bin/google-chrome` by default.

**Comment by wk-done** [comment]
Hi auchenberg,

thanks for repyling and for the hints.
I tried `which chromium-browser` and the result was `/usr/bin/chromium-browser`.
I set the value for `runtimeExecutable` and **it works**.
Here the complete config:
```json
{
  "name": "ng serve",
  "runtimeExecutable": "/usr/bin/chromium-browser",
  "type": "chrome",
  "request": "launch",
  "url": "http://localhost:4200/#",
  "webRoot": "${workspaceRoot}"
}
```
But I tried the same yesterday and it did not work. (See my command above).
Anyways, your hints are very helpful because now I could define an alias, a symlink or something like this.
Thank you very much.
